### PR TITLE
Add specs for testing labelField & uniqueIdField

### LINKED ISSUE #76: Add specs for testing labelField & uniqueIdField
As of #75 (which was a `cherry-pick` of #53), we will begin allowing users to set:
1.  a `labelField` prop, to identify which property on their options should be used for display
2. a `uniqueIdField` prop, to identify which property on their options is guaranteed to be unique, and thus can be used for tracking selected options

We now need to ensure there are functional specs in place for those props

### PR DESCRIPTION
**Pull request by jackkoppa**
Closes #76 

Currently PRing into the branch where these props are added, to make review of each step easier

### CODE REVIEW

File: specs/SelectMulti.spec.ts [hunk 1]
```diff
@@ -1,66 +1,136 @@
...
-				{ label: 'Item Three', value: 'three' }
-			]
+		beforeEach(() => options = simpleOptions )
 
-			wrapper = mount({
-				data() { 
-					return { selectedOptions: [], options }},
-				template: '<div><SelectMulti :options="options" v-model="selectedOptions" label="Sample SelectMulti" /></div>',
-				components: { SelectMulti }
+		describe('and a simple SelectMulti initialized with options, v-model, and a label', () => {
```
Comment (EvanSanderson) [inline specs/SelectMulti.spec.ts:18]: And by "simple" we mean "does not have custom labels or unique id fields beyond the value property" correct? Just want to make sure i understand the nomenclature

File: specs/sampleData.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    },
+    { 
+        customLabel: 'Item Two',
+        customUniqueId: 'two' 
+    },
+    { 
+        customLabel: 'Item Three',
+        customUniqueId: 'three'
+    }
+]
```
Comment (EvanSanderson) [inline specs/sampleData.ts:22]: Do we want one additional case where we have { label: 'something', value: 'something', customUniqueId: 'something-else'}? Is that worth it?

**Review by EvanSanderson — COMMENTED** [review]
Really really well organized and easy to follow

File: specs/SelectMulti.spec.ts (hunk 1 above)
Comment (jackkoppa) [reply specs/SelectMulti.spec.ts:18]: Yeah, I wonder if I can word it better. What I mean is "these are the bare minimum props for the component to work at all", so it's actually not specific to missing `labelField` or `uniqueIdField`. I'm gonna do:

`and a SelectMulti initialized with the bare minimum props: options, v-model, and a label`

if that sounds good?

File: specs/sampleData.ts (hunk 2 above)
Comment (jackkoppa) [reply specs/sampleData.ts:22]: Absolutely, I think it is. To show that that's totally valid - the `value` in that case isn't cared about by our package at all, but will of course still exist on the objects when emitted, etc.

So yeah, I think that's worth showing. Will do

**Comment by jackkoppa** [comment]
Howdy @wickkidd et al. - I had created this PR right around the time I was finishing up at Politico. It was... maybe helpful at the time, but now that I'm looking at my GitHub PRs more regularly again, I'm noticing it was never merged. Given I assume it's not causing an active issue for you, I think I'll go ahead & close to not break anything - the commits will still be around for future reference, obviously, in case something similar ends up being useful later. Doing the same for this other stale PR:

https://github.com/politico/vue-accessible-selects/pull/79

**Pull request closed without merging**